buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -554,11 +554,11 @@ class BeamModulePlugin implements Plugin<Project> {
     def log4j2_version = "2.20.0"
     def nemo_version = "0.1"
     // Try to keep netty_version consistent with the netty version in grpc_bom (includes grpc_netty) in google_cloud_platform_libraries_bom
-    def netty_version = "4.1.87.Final"
+    def netty_version = "4.1.96.Final"
     def postgres_version = "42.2.16"
     def powermock_version = "2.0.9"
     // Try to keep protobuf_version consistent with the protobuf version in google_cloud_platform_libraries_bom
-    def protobuf_version = "3.23.2"
+    def protobuf_version = "3.24.0"
```

Comment:
I was basing this on https://mvnrepository.com/artifact/com.google.cloud/libraries-bom/26.22.0 . Where are you getting the dependencies from?

Reply:
I create a POM that depends on all versions from the libraries-bom

```
wget https://repo1.maven.org/maven2/com/google/cloud/libraries-bom/26.22.0/libraries-bom-26.22.0.pom -O base.pom && mvn help:effective-pom -f base.pom -Doutput=effective.pom && cat effective.pom | grep -v 'dependencyManagement' > cleanup.pom && mvn dependency:tree -f cleanup.pom
```

And then I look at the left/top-most dependencies in the tree, since Maven uses distance for conflict resolution.

Reply:
ok, reverted